src/cascadia/TerminalApp/TerminalTab.cpp

```diff
@@ -1763,4 +1769,88 @@ namespace winrt::TerminalApp::implementation
 
         return Title();
     }
+
+    // Method Description:
+    // - Toggle broadcasting input to all the panes in this tab.
+    void TerminalTab::ToggleBroadcastInput()
+    {
+        _tabStatus.IsInputBroadcastActive(!_tabStatus.IsInputBroadcastActive());
+
+        const bool isBroadcasting = _tabStatus.IsInputBroadcastActive();
+        _rootPane->EnableBroadcast(isBroadcasting);

```

Comment:
nit: fold or reverse?

```suggestion
        const bool newIsBroadcasting = !_tabStatus.IsInputBroadcastActive();
        _tabStatus.IsInputBroadcastActive(newIsBroadcasting);
        _rootPane->EnableBroadcast(newIsBroadcasting);
```

("Set on X and then ask X" weirds me out)